atom/browser/api/atom_api_desktop_capturer.cc

```diff
@@ -58,17 +58,11 @@ DesktopCapturer::~DesktopCapturer() {}
 void DesktopCapturer::StartHandling(bool capture_window,
                                     bool capture_screen,
                                     const gfx::Size& thumbnail_size) {
-  webrtc::DesktopCaptureOptions options =
-      content::desktop_capture::CreateDesktopCaptureOptions();
-#if defined(OS_WIN)
-  using_directx_capturer_ = options.allow_directx_capturer();
```

Comment:
@nitsakh Are you sure this should be removed?

Reply:
Yeah, chromium does that. https://cs.chromium.org/chromium/src/content/public/browser/desktop_capture.cc?rcl=92e0446435896425805d05af8a91072263223d68&l=14

Reply:
But we use `using_directx_capturer_ ` here
https://github.com/electron/electron/blob/bc2387cf1c02182c47771a65d69c1a6ff946a172/atom/browser/api/atom_api_desktop_capturer.cc#L97